src/test/java/com/oscarcommerce/utils/User.java

```diff
@@ -0,0 +1,67 @@
+package com.oscarcommerce.utils;
+
+public class User {
+
+    private ShippingAddress shippingAddress;
```

Comment:
As we discussed, it's worth to create a separate class Shipping Address if we store a List within User. Lets refactor it together.